tests/TestApp/AdalDesktopTestApp/Program.cs

```diff
@@ -54,11 +59,66 @@ static void Main(string[] args)
 
         private static async Task AcquireTokenAsync()
         {
-            AuthenticationContext context = new AuthenticationContext("https://login.microsoftonline.com/common", true);
-            var result = await context.AcquireTokenAsync("https://graph.windows.net", "<CLIENT_ID>", new UserCredential("<user>"));
+            Telemetry telemetry = Telemetry.GetInstance();
+            DispatcherImplement dispatcher = new DispatcherImplement();
+            telemetry.RegisterDispatcher(dispatcher, false);
+
+            //LoggerCallbackHandler.Callback = new MyCallback();
```

Comment:
>//LoggerCallbackHandler.Callback = new MyCallback(); [](start = 12, length = 52)

remove commented code. if callback is not being used then remove the implementation class as well.